### PR TITLE
docs/opentelemetry: adds opentemetry get started guide

### PR DESCRIPTION
**Pull request by f41gh7**
it must clarify usage of openetelemetry metrics format with VictoriaMetrics components

### CODE REVIEW

File: docs/guides/getting-started-with-opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+# at url with browser http://localhost:8428/vmui/#/?g0.expr=k8s_container_ready
+```
+
+  Full version of possible configuration options could be found at [docs](https://opentelemetry.io/docs/collector/configuration/)
+
+## Direct metrics push
+ Metrics could be ingested into VictoriaMetrics directly with http requests. You can use any compatible opentelemetry instrumentation [clients](https://opentelemetry.io/docs/languages/).
+In our example, we'll create golang webserver and instrument metrics for it.
+
+### create sample apple
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:92]: i guess you need to rename this

File: docs/guides/getting-started-with-opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
+---
+weight: 5
+title: How to use opentelemetry metrics with VictoriaMetrics
+menu:
+  docs:
+    parent: "guides"
+    weight: 5
+---
+
+# Application monitoring with opentelemetry
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:10]: can we add the image something like 

[otel example.excalidraw.zip](https://github.com/VictoriaMetrics/VictoriaMetrics/files/14397988/otel.example.excalidraw.zip)

File: docs/guides/getting-started-with-opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+#### test metrics ingestion 
+
+```sh
+# port-forward victoriametrics to ingest metrics
+kubectl port-forward victoria-metrics-victoria-metrics-single-server-0 8428
+
+# open vmui and query `http_requests_total` and `http_request_latency_seconds_bucket`
+```
+
+## Known opentelemetry limitations.
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:373]: just limitations

File: docs/guides/getting-started-with-opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+		return nil, fmt.Errorf("cannot Register int64 gauge: %w", err)
+	}
+	mw.onShutdown = mc.Stop
+
+	return mw, nil
+}
+```
+### full app example
+
+```golang
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:185]: golang -> go

File: docs/guides/getting-started-with-opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,377 @@
...
+		ar.Observe(ctx, atomic.LoadInt64(&mw.activeRequests))
+	}); err != nil {
+		return nil, fmt.Errorf("cannot Register int64 gauge: %w", err)
+	}
+	mw.onShutdown = mc.Stop
+
+	return mw, nil
+}
+```
+### full app example
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:183]: can we have a new line after and before every  ```

File: docs/guides/getting-started-with-opentelemetry.md [hunk 6]
```diff
@@ -0,0 +1,377 @@
...
+
+VictoriaMetrics supports metrics ingestion with [opentelemetry](https://opentelemetry.io/docs/specs/otel/metrics/) metrics format.
+This guide covers data ingestion with [opentelemetry-colletor](https://opentelemetry.io/docs/collector/) and direct metrics push from application.
+
+## Pre-Requirements  
+
+ * kubernenetes cluster
+ * kubectl
+ * helm
+### Install VictoriaMetrics single with helm chart
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:20]: new line is missing

File: docs/guides/getting-started-with-opentelemetry.md [hunk 7]
```diff
@@ -0,0 +1,377 @@
...
+title: How to use opentelemetry metrics with VictoriaMetrics
+menu:
+  docs:
+    parent: "guides"
+    weight: 5
+---
+
+# Application monitoring with opentelemetry
+
+VictoriaMetrics supports metrics ingestion with [opentelemetry](https://opentelemetry.io/docs/specs/otel/metrics/) metrics format.
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:12]: opentelemetry -> OpenTelemetry

File: docs/guides/getting-started-with-opentelemetry.md [hunk 8]
```diff
@@ -0,0 +1,377 @@
...
+This guide covers data ingestion with [opentelemetry-colletor](https://opentelemetry.io/docs/collector/) and direct metrics push from application.
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:13]: it covers more :)

File: docs/guides/getting-started-with-opentelemetry.md [hunk 9]
```diff
@@ -0,0 +1,377 @@
...
+
+## Pre-Requirements  
+
+ * kubernenetes cluster
+ * kubectl
+ * helm
+### Install VictoriaMetrics single with helm chart 
+
+Install single node version:
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:22]: Adding VictoriaMetrics helm repo and install VictoriaMetrcsi Single-Node ...

File: docs/guides/getting-started-with-opentelemetry.md [hunk 10]
```diff
@@ -0,0 +1,377 @@
+---
+weight: 5
+title: How to use opentelemetry metrics with VictoriaMetrics
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:3]: opentelemetry -> OpenTelemetry

File: docs/guides/getting-started-with-opentelemetry.md [hunk 11]
```diff
@@ -0,0 +1,377 @@
...
+In our example, we'll create golang webserver and instrument metrics for it.
+
+### create sample apple
+At first we have to define go.mod with app dependecies:
+```golang
+module otlp-web-example
+
+go 1.18
+
+require (
```
Comment (tenmozes) [inline docs/guides/getting-started-with-opentelemetry.md:99]: should we have somewhere code example?

**Review by tenmozes — COMMENTED** [review]
Thank you so much for the guide! it's awesome

File: docs/guides/getting-started-with-opentelemetry.md (hunk 10 above)
Comment (AndrewChubatiuk) [reply docs/guides/getting-started-with-opentelemetry.md:3]: can add information about spanmetrics processor, which allows to convert traces to metrics as a temporary workaround for users who wants use VM for traces as well
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/connector/spanmetricsconnector/README.md

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Pull request merged**